userops_ext.go

```diff
@@ -595,3 +844,64 @@ func (op *UserOperation) String() string {
 		formatBytes(op.Signature),
 	)
 }
+
+// encodeCrossChainCallData encodes the call data in the cross-chain format
+// [2 bytes opType (0xFFFF)] + [2 bytes callDataLength] + [callData] + [1 byte hashListLength] + [Hash List Entries]
+func (op *UserOperation) encodeCrossChainCallData(callData []byte) ([]byte, error) {
```

Comment:
This function (encodeCrossChainCallData) is done only by the SDK. Please move it to the test file if you need it for mocking.

Reply:
oh the CLI will do this besides the SDK...